CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `ean` on `CartItem`.
```

Comment:
No need for the changelog or release a new version. 

Just adding to the types because this app is sometimes used as an example.

```suggestion
```